openssl/src/bn/mod.rs

```diff
@@ -473,6 +473,11 @@ impl BigNum {
         n
     }
 
+    pub fn into_raw(self) -> *mut ffi::BIGNUM {
```

Comment:
Doesn't need to be fixed, but you can just do `pub fn into_raw(mut self)` and avoid the `let mut me = self` line.